sql/tables.sql

```diff
@@ -119,10 +119,17 @@ SELECT pg_catalog.pg_extension_config_dump(pg_get_serial_sequence('_timescaledb_
 -- on the chunk's data table.
 CREATE TABLE _timescaledb_catalog.chunk_constraint (
     chunk_id            INTEGER  NOT NULL REFERENCES _timescaledb_catalog.chunk(id) ON DELETE CASCADE,
-    dimension_slice_id  INTEGER  NOT NULL REFERENCES _timescaledb_catalog.dimension_slice(id) ON DELETE CASCADE,
-    PRIMARY KEY(chunk_id, dimension_slice_id)
+    dimension_slice_id  INTEGER  NULL REFERENCES _timescaledb_catalog.dimension_slice(id) ON DELETE CASCADE,
+    constraint_name     NAME NOT NULL,
+    hypertable_constraint_name NAME NULL,
+    PRIMARY KEY(chunk_id, constraint_name)
```

Comment:
I think primary keys in metadata tables should be OIDs (or SERIALs) so that we don't have CASCADEs to other tables  in case of changing, e.g., a name that is part of a primary key. (We had this discussion earlier and I think there was agreement to simplify metadata tables this way and align with pg_catalog). 

Strictly, I don't think this table needs a PRIMARY KEY, so maybe just convert this to a UNIQUE constraint? If it needs a PRIMARY KEY, we should add a SERIAL id.

Reply:
Fixed, made unique constraint (In PR #198)